index.js

```diff
@@ -0,0 +1,37 @@
+    document.addEventListener('DOMContentLoaded', () => {
+        
+        const form = document.querySelector('form');
+        const displaySection = document.querySelector('.collectNotes');
+
+        form.addEventListener('submit', (event) => {
+        event.preventDefault();
+    
+        const formData = new FormData(form);
+        const sanitizedData = {};
+
+        formData.forEach((value, key) => {
```

Comment:
❓Are u able to explain this block of code with your own words? 🤔 ...with your own words.